src/docs/development/tools/android-studio.md

```diff
@@ -159,45 +161,82 @@ Also available in the "more actions" menu:
+The exact count of the rebuilds for this frame displays in the second
+column from the right. For a high number of rebuilds, a yellow spinning
+circle displays. The column to the far right shows how many times a
+widget was rebuilt since entering the current screen.
+The app shown in this screenshot demonstrates poor performance.
+For widgets that aren't rebuilt, a solid grey circle displays.
 Otherwise, a grey spinning circle displays.
 
-### Debugging with Observatory
+The purpose of this tool is to make you aware when widgets are
+rebuilding&mdash;you might not realize that this is happening when just
+looking at the code. If widgets are rebuilding that you didn't expect,
+it's probably a sign that you should refactor your code by splitting
+up large build methods into multiple widgets.
+
+This tool can help you detect at least four common performance issues:
+
+1. The whole screen (or large pieces of it) are built by a single
+   StatefulWidget, causing unnecessary UI building.
+
+1. Offscreen widgets are being rebuilt. This can happen, for example,
+   when a ListView is nested in a tall Column that extends offscreen.
+   Or when the RepaintBoundary is not set for a list that extends
+   offscreen, causing the whole list to be redrawn.
+
+1. The `build()` function for an AnimatedBuilder draws a subtree that
+   does not need to be animated, causing unnecessary rebuilds of static
+   objects.
 
-Observatory is an additional debugging and profiling tool presented with an
-html-based UI. For details see the [Observatory
-page](https://dart-lang.github.io/observatory/).
+1. An Opacity widget is placed unnecessarily high in the widget tree.
+   Or, an Opacity animation is created by directly manipulating the
+   opacity property of the Opacity widget, causing the widget itself
+   and its subtree to rebuild.
 
-To open Observatory:
+You can click on a line in the table to navigate to the line
+in the source where the widget is created. As the code runs,
+the spinning icons also display in the code pane to help you
+visualize which rebuilds are happening.
 
- 1. Run your app in debug mode.
- 1. Select the **Open Observatory** action from the Debug panel.
- 1. Click the **Stopwatch icon**:<br>
-    ![Debugging panel]({% asset tools/android-studio/debug-panel.png @path %}){:width="700px"}
+Note that numerous rebuilds doesn't necessarily indicate a problem.
+Typically you should only worry about excessive rebuilds if you have 
+already run the app in profile mode and verified that the performance
+is not what you want.
+
+And remember, _the widget rebuild information is only available in
+a debug build_. Test the app's performance on a real device in a profile
+build, but debug performance issues in a debug build.
+
+### Debugging with Dart DevTools
+
+Dart DevTools are a set of debugging and profiling tools presented
+with an html-based UI. DevTools replaces the previous
+browser-based profiling tool, [Observatory][].
```

Comment:
ref still points to [Observatory]
Remove all references to [Observatory] if you can.